knimin/tests/test_ag_pulldown.py

```diff
@@ -73,6 +73,11 @@ def test_post(self):
                        ' for file download. It may take a while with many '
                        'barcodes.</h3>'), response.body)
 
+        data = {'external': 'ab', 'external': 'cd'}
```

Comment:
sorry @josenavas but I think it is not passing a list of strings to the method. What I get is a `u'["foo", "bar"]`, i.e. the string representation of a list of strings :-/